src/components/Swap/Toolbar/Caption.tsx

```diff
@@ -180,17 +200,29 @@ export interface TradeProps {
   outputUSDC?: CurrencyAmount<Currency>
 }
 
-export function Trade({ trade, outputUSDC, gasUseEstimateUSD }: TradeProps & GasEstimateProps) {
+interface ExpandProps {
+  expanded: boolean
+  toggleExpanded: () => void
```

Comment:
This is a handler, so it should be called `on*`. `onExpand` or `onToggleExpand`?